.github/workflows/cli.yaml

```diff
@@ -41,3 +42,178 @@ jobs:
+
+            - name: Building tarballs
+              run: yarn oclif pack tarballs --root="./apps/cli"
+
+            - name: Upload tarball packages to Github artifacts
+              uses: actions/upload-artifact@v3
+              with:
+                  name: Packages-tarballs
+                  path: ./apps/cli/dist/*.tar.*
+
+            - name: Upload tarball packages to S3
+              working-directory: apps/cli
+              run: |
+                yarn oclif upload tarballs
+
+    promote-deb-and-tarballs:
+      needs: [pack_upload_deb, pack_upload_tarballs]
+      runs-on: ubuntu-latest
+      steps:
+            - uses: actions/checkout@v3
+
+            - name: Install system deps
+              run: |
+                sudo apt-get update
+                sudo apt-get install -y nsis p7zip-full
+
+            - name: Setup Node.js 20
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 20
+                  cache: "yarn"
+
+            - name: Install Foundry
+              uses: foundry-rs/foundry-toolchain@v1
+
+            - name: Install dependencies
+              run: yarn install
+
+            - name: Build
+              run: yarn build --filter @sunodo/cli
+
+            - name: Install oclif
+              run: yarn global add oclif
+
+            - name: Configure AWS Credentials
+              uses: aws-actions/configure-aws-credentials@v2
+              with:
+                  role-to-assume: ${{ secrets.IAM_ROLE_ARN }}
+                  aws-region: ${{ secrets.AWS_REGION }}
+
+            - name: Download deb packages from Github artifacts
+              uses: actions/download-artifact@v3
+              with:
+                name: Packages-deb
+                path: /home/runner/work/sunodo/sunodo/apps/cli/dist
+
+            - name: List all the downloaded files (for debugging)
+              run: ls -R
+              working-directory: /home/runner/work/sunodo/sunodo/apps/cli/dist
+
+            - name: Extract version and SHA
+              id: extract_info
+              run: |
+                filename=$(find . -name '*amd64.deb' -type f)
+                version=$(echo "$filename" | sed -n 's/.*_\([0-9]*\.[0-9]*\.[0-9]*\).*-.*\.deb/\1/p')
+                sha=$(echo "$filename" | sed -n 's/.*\.\([^-]*\)-.*\.deb/\1/p')
+
+                echo "::set-output name=version::$version"
+                echo "::set-output name=sha::$sha"
+              working-directory: apps/cli/dist
+
+            - name: Promote packages to S3
+              run: |
+                yarn oclif promote --deb --xz --indexes --version=${{ steps.extract_info.outputs.version }} --sha=${{ steps.extract_info.outputs.sha }}
+                aws s3 cp  s3://cli-prod-sunodo-origin/versions/${{ steps.extract_info.outputs.version }}/${{ steps.extract_info.outputs.sha }}/apt/./ s3://cli-prod-sunodo-origin/channels/stable/apt/./ --recursive
```

Comment:
do we need the extra `./` in those URLs?

Reply:
Yes, we need to add `./` contents into `./` of the stable channel.